src/environment.go

```diff
@@ -399,15 +399,42 @@ func (env *environment) getBatteryInfo() ([]*battery.Battery, error) {
 			validBatteries = append(validBatteries, batt)
 		}
 	}
+	// clean minor errors
 	unableToRetrieveBatteryInfo := "A device which does not exist was specified."
+	unknownChargeRate := "Unknown value received"
+	var fatalErr battery.Errors
+	ignoreErr := func(err error) error {
```

Comment:
Why not return a bool? Saves some space below, you already have the error.

Reply:
Good point =D I copied this from the battery lib itself and changed it a little bit to fit this case. But a bool makes more sense.

Reply:
@JanDeDobbeleer  Done